src/data/actions/courseInfo.js

```diff
@@ -174,6 +180,32 @@ function createCourse(courseData) {
   };
 }
 
+function handleCourseRuns(dispatch, courseRunData, course, submitReview) {
+  // status property will be present for internal review process because we pass the run
```

Comment:
nit: Clean up this comment's wording so it clarifies the functionality more quickly.
```We'll use the presence of the course run status property to signal an internal review process because it is only present when we pass in a single run for internal review, rather than an array of runs on a regular edit```

Reply:
will change now :)